Assets/Package/Runtime/Console/ConsoleGui.cs

```diff
@@ -144,14 +161,34 @@ private void EnsureInitialized()
             {
                 { inputSettings.ConfirmationKey, InvokeInputString },
                 { inputSettings.NextCommandKey, FetchNextCommand },
-                { inputSettings.PrevCommandKey, FetchPrevCommand }
+                { inputSettings.PrevCommandKey, FetchPrevCommand },
+                { inputSettings.AutocompleteKey, PerformAutocomplete }
             };
 
+            autocompleteHandler = new AutocompleteHandler();
             windowRect = styleSettings.InitialRect;
             scrollPosition = Vector2.zero;
             isInitialized = true;
         }
 
+        private void InputChanged()
+        {
+            UpdateAutocomplete();
```

Comment:
Imho tej funkcji może nie być bo wywołuje tylko jedną metodę, czyli równie dobrze może jej nie być

Reply:
Trochę tak, ale też to rozwiązanie dosyć fajnie opisuje w którym momencie zadzieje się UpdateAutocomplete() bez dodatkowej analizy kodu. Moim zdaniem trochę czytelniej. (tak wiem, yagni, ale od razu mamy miejsce na wrzucanie rzeczy dziejących się po zmianie inputu)

Reply:
Imho jeżeli będziemy mieli taki przypadek to zmergujemy je wtedy do osobnej metody. Szczególnie, że metoda jest prywatna a nie publiczna to wygląda jakbyś ukrywał swój kod co go według mnie zaciemnia.

Reply:
IMO może zostać ale zmieniłbym nazwę na OnInputChange

Reply:
A jakie jest Twoje IMO Ludwiczku?